react-native/package-dependencies/handle-dependencies-with-yarn-override.mo.md

```diff
@@ -0,0 +1,75 @@
+# [MO] Handle dependencies with Yarn override
+
+## Owner: [Justine Mignot](https://github.com/justinemignot)
+
+> **Note**: Please create an [issue](https://github.com/bamlab/dev-standards/issues/new) or even a Pull Request with your feedback, typo correction.
+
+## Context
+
+During this standard, we will go through how to handle versions dependencies of different packages to a same package.
+
+If some package doesn't use same versions of a package, depending of the order it will be called, it won't be the same version of package used. We want to ensure that the same version is used everywhere in the app. Normally this would require to manually edit `yarn.lock` file, but by chance, yarn has a feature override, which allows us to fix those versions dependencies.
```

Comment:
Why is editing yarn.lock bad => deleted when installing new package or running yarn commands